<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>plotWindow</class>
 <widget class="QDialog" name="plotWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>850</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Plotting</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0" rowspan="2" colspan="2">
    <widget class="QCustomPlot" name="plot" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>1</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <widget class="QLineEdit" name="inputLine">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>10</y>
        <width>544</width>
        <height>21</height>
       </rect>
      </property>
      <property name="maxLength">
       <number>255</number>
      </property>
     </widget>
     <widget class="QPushButton" name="plotButton">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>5</y>
        <width>156</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>plot</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="xMin">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>65</y>
        <width>100</width>
        <height>21</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>x min value</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="xMax">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>65</y>
        <width>100</width>
        <height>21</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>x max value</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="yMin">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>95</y>
        <width>100</width>
        <height>21</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>y min value</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="yMax">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>95</y>
        <width>100</width>
        <height>21</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>y max value</string>
      </property>
     </widget>
     <widget class="QLabel" name="errorLine">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>35</y>
        <width>541</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
